src/runtime_src/core/pcie/driver/linux/xocl/subdev/qdma.c

```diff
@@ -1715,7 +1727,7 @@ static int qdma_probe(struct platform_device *pdev)
 	conf->qsets_max = QDMA_QSETS_MAX;
 	conf->bar_num_config = dma_bar;
 	conf->bar_num_stm = XDEV(xdev)->bar_idx;
-	conf->stm_reg_base = 0x02000000;
+	conf->stm_reg_base = stm_base;
```

Comment:
please use xocl_ioaddr_to_baroff() to get stm bar index. you may reference dma_bar

Reply:
changed. verified u200 legacy qdma